### ISSUE TITLE
Add 3 more colors, update existing 8 colors

### ISSUE THREAD

**Issue by ninavizz** [QubesOS/qubes-issues#6752]
**The problem you're addressing (if any)**

Take an incremental step towards expanding the available colors in Qubes.

**Describe the solution you'd like**
Per https://github.com/QubesOS/qubes-issues/issues/2523#issuecomment-837602318 in the original "More colors, pls!" Issue, I'd like to propose as the actionable-step in this issue:
- Add Magenta, Cyan, Lime from below
  - <strike>Intentionally choosing one from each of the three tone-tiers in the below; lime, magenta, and cyan, would be my personal pref. User feedback feels important, by pushing one from each of the 3 tiers into production.</strike>
  - Intentionally choosing colors far apart from each other in hue to add more increments on the ROYGBV security-scale.
- Adjust other/existing colors, to perceptually match the below values

<img width="708" alt="117743941-a11e0800-b1bc-11eb-884c-33c5e29ea379" src="https://user-images.githubusercontent.com/8262612/124015457-8c2e4b80-d999-11eb-9264-e6de8050309b.png">

**Where is the value to a user, and who might that user be?**

1. Many users have requested more colors.
2. Incremental rollouts of more colors is advised, over a tidal-wave of "all at once" for the following reasons:
  - More research around _how_ users use color, would be good.
  - A different selection mechanism than a straight drop-down is advised when more than 5 items exist in a menu (cough, a ux best practice commonly violated, heh)—and determining/implementing that would be a separate issue.
  - Creating new icons and testing new colors, is a lot of work.

**Describe alternatives you've considered**

Allowing users to "create" their own colors. Not a good idea, for many reasons—namely, that a) For contrast, "shading/darkening" of darker colors on icons is not a formulaic process, and b) MySpace reminiscent trainwreck opportunity of users creating their own unusable environments, by not understanding color and usability concepts. With color it is easy to "decorate," and in the process to impede one's ability to use a thing—without knowing that's what's happening.

**Additional context**

n/a

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**

n/a

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**

#2523, #6463

**Comment by andrewdavidwong** [comment]
> Per #2523 (comment) in the original "More colors, pls!" Issue

Isn't this a duplicate of that issue?

**Comment by ninavizz** [comment]
@andrewdavidwong I don't think so. I see a total of 10 colors in the above palette, that could be added. I'd like to still consider the other 7, but want to see what happens when we add these first three. I also don't want to comment any further on what became a veeeeery long, discussion-rich long issue. It's been discussed to death and just needs to either happen or not. Breaking-out this one piece as the clearly actionable next-step item seems like a good first step towards resolving the full issue. 

Does that make sense to you?

**Comment by andrewdavidwong** [comment]
> @andrewdavidwong I don't think so. I see a total of 10 colors in the above palette, that could be added. I'd like to still consider the other 7, but want to see what happens when we add these first three. I also don't want to comment any further on what became a veeeeery long, discussion-rich long issue. It's been discussed to death and just needs to either happen or not. Breaking-out this one piece as the clearly actionable next-step item seems like a good first step towards resolving the full issue. 
> 
> 
> 
> Does that make sense to you? 

I see this as just partially completing (i.e., making progress on) that issue, but it's fine to reorganize them in this way (this issue for three colors, the existing issue now for the remaining seven) if that makes sense for the work.

**Comment by ninavizz** [comment]
Labeled this issue "Help Wanted," because none of the core Qubes OS team are likely to be able to prioritize this anytime soon, abreast critical project priorities. That said: to me, this feels like an helpful addition to the UI. It's definitely one the community has been vocal in their asks for.

The core Qubes team have given their blessing to this UX solution, so should a developer like to take this on pls "@" me here and I'll hop-to gathering final SVG artwork to make that possible.

**Comment by airelemental** [comment]
IMHO some titlebar colors have less contrast in 4.1 than in 4.0.

Theme: Nodoka

4.0 had better contrast:
4.0 ![blue](https://user-images.githubusercontent.com/40210726/136884527-e6f05979-7eb7-4f38-ae3c-c69b065acb23.png)
4.1 ![blue](https://user-images.githubusercontent.com/40210726/136884548-b5d15ee8-7447-41a9-b8b3-e01af99714fe.png)
4.0 ![red](https://user-images.githubusercontent.com/40210726/136884920-fb6c7ace-5dc5-4d09-9555-2a2548d4cde0.png)
4.1 ![red](https://user-images.githubusercontent.com/40210726/136884860-0766beaa-c9f2-4167-aaa1-c69febda0a76.png)
4.0 ![orange](https://user-images.githubusercontent.com/40210726/136884794-d7a0c86d-e604-4d02-a9c1-eec8da24dfbf.png)
4.1 ![orange](https://user-images.githubusercontent.com/40210726/136884833-4d186f7b-b7a2-40a3-8912-ab45e84bbe63.png)

4.1 has better contrast:
4.0 ![gray](https://user-images.githubusercontent.com/40210726/136884555-3739a20b-5352-43e2-b887-3ddfbeee181e.png)
4.1 ![gray](https://user-images.githubusercontent.com/40210726/136884596-6ccf600c-3969-4e5e-ab71-f1878d710c97.png)
4.0 ![green](https://user-images.githubusercontent.com/40210726/136884660-c26a55f5-6f00-4019-9481-a2ece75a574b.png)
4.1 ![green](https://user-images.githubusercontent.com/40210726/136884683-b6397773-3714-49d2-98f0-07cd8e48294d.png)
4.0 ![yellow](https://user-images.githubusercontent.com/40210726/136884950-f262b327-2f03-4e7c-ad2c-c459457240d6.png)
4.1 ![yellow](https://user-images.githubusercontent.com/40210726/136884995-698ddb31-2f03-4bf6-9de1-56d10650fb28.png)
 
Equally good:
4.0 ![purple](https://user-images.githubusercontent.com/40210726/136885388-83d34a5e-6e18-4693-a729-9985e4f111b5.png)
4.1 ![purple](https://user-images.githubusercontent.com/40210726/136885415-8cb8e7ca-24f3-499a-8642-cf087a745d4c.png)
4.0 ![black](https://user-images.githubusercontent.com/40210726/136884435-687cb023-bd59-42bb-921a-77a3e21e6439.png)
4.1 ![black](https://user-images.githubusercontent.com/40210726/136885340-1b85cef2-7299-4632-bb0c-76b852f7fadc.png)

**Comment by ninavizz** [comment]
A quick (quick!) stab at how this ticket would edit and extend the colors w/in the qute qubes family:
<img width="660" alt="Screen Shot 2021-11-09 at 1 49 38 PM" src="https://user-images.githubusercontent.com/8262612/141010713-8ee69456-326d-4aa1-bd17-cb3bb5aa5399.png">

**Comment by mdik** [comment]
What is blocking this issue from being closed?

**Comment by ben-grande** [comment]
> What is blocking this issue from being closed?

It being done or cancelled.
